.github/workflows/blank.yml

```diff
@@ -0,0 +1,17 @@
+name: Node.js CI
```

Comment:
Why is it called blank?

Reply:
I created through a blank template on github UI. I tried renaming it but it seem to stop the process. I'll try again later